plugins/modules/vmware_dvswitch_info.py

```diff
@@ -268,8 +268,10 @@ def all_info(self):
                         'healthCheck': health_check
                     },
                     'hosts': host_members,
-                    'folder': switch_obj.parent.name
-                }
+                    'folder': switch_obj.parent.name,
+                    'name': switch_obj.name,
+                },
+                'uuid': switch_obj.uuid,
```

Comment:
Could you please add the `uuid` key to RETURN block?

Reply:
The whole dict is already in the result list. The UUID, is immutable, so I don't it should be a subkey of `configuration`.

Reply:
Do not need `uuid` in sample?
I thought it would be easier to understand if there was a `uuid` in the sample.

```
RETURN = '''
(snip)
    sample:
      [
        {
          "configure": {
          (snip)
              },
              "privateVlan": []
            }
          },
          "uuid": "50 30 99 9c a7 60 8a 4f-05 9f e7 b5 da df 8f 17" <- sample
        }
      ]
'''
```

Reply:
oh my bad. You're absolutely right.